src/store/messagesStore.js

```diff
@@ -521,10 +522,14 @@ const actions = {
 			const parentInStore = context.getters.message(message.token, message.parent.id)
 			if (Object.keys(parentInStore).length !== 0) {
 				context.commit('addMessage', message.parent)
-				context.dispatch('resetReactions', {
-					token: message.token,
-					messageId: message.parent.id,
-				})
+				const reactionsStore = useReactionsStore()
+				if (message.actorId !== context.getters.getActorId()) {
+					if (message.systemMessage.startsWith('reaction')) {
+						reactionsStore.fetchReactions(message.token, message.parent.id)
```

Comment:
I'm afraid, that is still to many requests we won't use at all.
As I see it, here we could:
- check if new system message contains reactions different than we have now in store (and we don't need to do anything)
- instead of resetting existing details, we can mark them as outdated instead of resetting
  - so we need to fetch them only next time you want to see details

Reply:
> check if new system message contains reactions different than we have now in store (and we don't need to do anything)

The problem is that revoke reaction sys msg doesn't have all needed info so far ( emoji is lacking). However, considering our available resources in 'reaction' sys msg ( it includes all we need), we can skip fetch request when we receive the msg which can really minimize requests as adding a reaction is most frequent. So, I can get exactly which actor in which reaction in which message Id I will add, even the timestamp is included. 

> instead of resetting existing details

Here, if you mean in this conditional statement, getting the else condition is only for 'message_deleted', which is needed as the message is deleted so its reactions should be deleted.